homeassistant/components/subaru/translations/en.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
This file shouldn't be committed. It will be created automatically.

Reply:
File removed.